pygmt/tests/test_inset.py

```diff
@@ -0,0 +1,42 @@
+"""
+Tests for the inset function.
+"""
+from pygmt import Figure
+from pygmt.helpers.testing import check_figures_equal
+
+
+@check_figures_equal()
+def test_inset_aliases():
+    """
+    Test the aliases for the inset function.
+    """
+    fig_ref, fig_test = Figure(), Figure()
+    fig_ref.basemap(region="MG+r2", frame="afg")
+    with fig_ref.inset(D="jTL+w3.5c+o0.2c", M=0, F="+pgreen"):
+        fig_ref.basemap(region="g", projection="G47/-20/4c", frame="afg")
```

Comment:
We should use single letters for the reference images.
```suggestion
    fig_ref.basemap(R="MG+r2", B="afg")
    with fig_ref.inset(D="jTL+w3.5c+o0.2c", M=0, F="+pgreen"):
        fig_ref.basemap(R="g", J="G47/-20/4c", B="afg")
```